examples/features/JumpTable.huff

```diff
@@ -0,0 +1,32 @@
+#define macro MAIN() = takes(0) returns(0) {
+    0x21 calldataload                // [y]
+    0x01 calldataload                // [y, x]
+    2                                // [y, x, 2]
+    0x0 calldataload 0x0 byte        // [y, x, 2, b]
+    0x1 shl                          // [y, x, 2, b * 2]
+    __tablestart(OPS) add            // [y, x, 2, label_offset]
```

Comment:
why does the table store the offsets rather than the actual dests? if it stored the actual dests, no need to add, and the table could be reused in multiple locations

Reply:
Wdym? There's no difference between offsets and dests here

Reply:
By offsets I mean adding the offset to the table start, whereas for labels you would not be offsetting from the tablestart. You use add here and forbid outside labels so I presume you are offsetting from the tablestart.

Reply:
oh nm I misread this